src/selenium/web_storage_manager.cr

```diff
@@ -0,0 +1,49 @@
+class Selenium::WebStorageManager
+  enum StorageType
+    LocalStorage
+    SessionStorage
+  end
+
+  getter document_manager : DocumentManager
+
+  forward_missing_to @document_manager
+
+  def initialize(@document_manager, @storage : StorageType = StorageType::LocalStorage)
+  end
+
+  # Clear all items from the Storage
+  def clear
+    document_manager.execute_script("#{storage_type_in_js}.clear()")
+  end
+
+  # Retrieve and insert item from/into the Storage.
+  # If no value is given, it will return the item specified in `name`.
+  # For a given `value`, it will be added or updated for the item specified in `name`.
+  def item(name : String, value = nil) : String | Nil
+    if value.nil?
+      document_manager.execute_script("return #{storage_type_in_js}.getItem('#{name}')")
```

Comment:
It could interesting to parse the returned result. It is always a String :confused: